Tensile/Source/lib/include/Tensile/AMDGPU.hpp

```diff
@@ -218,6 +222,27 @@ namespace Tensile
 
         virtual std::string description() const;
 
+        const int getSKDynamicGrid() const
+        {
+            static const char* envStr = std::getenv("TENSILE_STREAMK_DYNAMIC_GRID");
```

Comment:
Where will you document these 3 env parameters?
Wiki? Common.py? Both?

Reply:
If we get better performance with these env variable and generate Tensile kernels based on these specific env values, users always need to specify these env values to get expected performance with StreamK kernels, right?
Can we really expect uses do that?

Reply:
I'm not sure of the best place to document this. I think I'll add some notes about them to Common.py for now.
These are added as env values because they are not properties of the stream-k kernel - any stream-k kernel can run with more or fewer workgroups, and there may be benefits or different use-cases for different modes. I don't know if there is an optimal way to use this yet, the team is still researching the effect of this feature. If there is a better default, we can also update the defaults later.
I think there will mainly be 2 modes for most users: the current default is to use 100% of the GPU for each kernel, which is the same as our other data-parallel kernels. The other mode would be dynamic, or to just limit the CU count so other operations can run in parallel. I think this mode would generally be set by frameworks that know they get faster results for larger workflows this way.
We still need to do a bunch of research and data collection to find the best way to use this. The env variables will let us do that testing, and it might show there's a better set of defaults, or that we need a different way of configuring these settings later.

Reply:
I think adding some comment in Common.py is OK for now.
I am OK to use env variables for experimental or debug purpose.
I do not recommend to use env variables for actual tuning because the perf number might not be reproducible without setting correct env variables.
Would you please add some comment to clarify this is only for experimental purpose and not for actual tuning?